src/inmanta/ast/statements/define.py

```diff
@@ -675,9 +606,9 @@ def evaluate(self) -> None:
         """
         Add the index to the entity
         """
-        entity_like = self.namespace.get_type(self.type)
-        assert isinstance(entity_like, EntityLike), "%s is not an entity or default" % entity_like
-        entity_type = entity_like.get_entity()
+        entity = self.namespace.get_type(self.type)
+        assert isinstance(entity, Entity), "%s is not an entity or default" % entity
+        entity_type = entity
```

Comment:
Might as well name it `entity_type` from the start.
```suggestion
        entity_type = self.namespace.get_type(self.type)
        assert isinstance(entity_type, Entity), "%s is not an entity or default" % entity_type
```